src/components/borrowers/Quotes.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { StaticImage } from 'gatsby-plugin-image';
+import { QuoteA55 } from './quotes/QuoteA55';
+import { QuoteTecredi } from './quotes/QuoteTecredi';
+
+export const Quotes = () => {
+  const [quote, setQuote] = React.useState('a55');
+
+  const handleClickQuote = (quote: string) => {
+    console.log('cliecked')
+    if (quote === 'a55') {
+      setQuote('a55')
+    }
+    if (quote === 'tecredi') {
+      setQuote('tecredi')
+    }
+  }
+
+  return (
+    <div className="h-flex-column quotes-component">
+      <div className="h-flex-row">
+        <span className="quotes-component--title">Borrowers</span>
+      </div>
+      <div className="h-flex-row h-flex-row--space-between quotes-component--text-row">
+        <div className="h-flex-column">
+          <span onClick={() => handleClickQuote('a55')}><StaticImage className="quotes-component--background-shape-a55" src="../../assets/logos/a55-logo.png" alt="a55 logo"/></span>
```

Comment:
just call setQuote here, and maybe define the value of "a55" as an enum

Reply:
oh and also add a 'h-cursor-pointer' class

Reply:
done